genprotocol/sserver-avdl/sserver.avdl

```diff
@@ -0,0 +1,7 @@
+@namespace("sserver")
```

Comment:
I think it's good practice to have a ".1" suffix in case we ever change it in the future, and have two versions of the protocol in the wild.